shared/common-adapters/tab-bar.desktop.js

```diff
@@ -90,12 +90,12 @@ class TabBarButton extends Component<void, TabBarButtonProps, void> {
     )
   }
 
-  _renderIcon (color: string, badgeNumber: number) {
-    if (this.props.source.type !== 'icon') return // needed to make flow happy
+  _renderDefault (color: string, badgeNumber: number) {
```

Comment:
I don't understand the point of this change. This function renders the icon variant. It just happens to be the default, but it shouldn't be defined as being the default.